install/aws-terraform/variables.tf

```diff
@@ -4,16 +4,19 @@
  */
 
 variable "project" {
-  type    = string
-  default = ""
+  type        = string
+  default     = "self-hosted"
+  description = "This is used in the naming of some resources"
 }
 
 variable "region" {
-  type = string
+  type        = string
+  description = "The AWS region that resources should be created in"
 }
 
 variable "domain" {
-  type = string
+  type        = string
+  description = "The domain that Gitpod should be installed on"
 }
 
 variable "chart_location" {
```

Comment:
That is no good path declaration. Please use the `path.root` or `path.cwd` variable.

Reply:
those variables are used in the script. However they cannot be used as default values in terraform

Reply:
He is referencing the chart path